test/e2e/basic_test.go

```diff
@@ -593,3 +593,50 @@ func TestDeletePreviousRoundFlowsOnStartup(t *testing.T) {
 		t.Errorf("Flow was still present after timeout")
 	}
 }
+
+// TestGratuitousARP verifies that we receive 3 GARP packets after a Pod is up.
+// There might be ARP packets other than GARP sent if there is any unintentional
+// traffic. So we just check the number of ARP packets is greater than 3.
+func TestGratuitousARP(t *testing.T) {
+	data, err := setupTest(t)
+	if err != nil {
+		t.Fatalf("Error when setting up test: %v", err)
+	}
+	defer teardownTest(t, data)
+
+	podName := "busybox"
+	nodeName := masterNodeName()
+
+	t.Logf("Creating Pod '%s' on '%s'", podName, nodeName)
+	if err := data.createBusyboxPodOnNode(podName, nodeName); err != nil {
+		t.Fatalf("Error when creating Pod '%s': %v", podName, err)
+	}
+	defer deletePodWrapper(t, data, podName)
+
+	antreaPodName, err := data.getAntreaPodOnNode(nodeName)
+	if err != nil {
+		t.Fatalf("Error when retrieving the name of the Antrea Pod running on Node '%s': %v", nodeName, err)
+	}
+
+	podIP, err := data.podWaitForIP(defaultTimeout, podName, testNamespace)
+	if err != nil {
+		t.Fatalf("Error when waiting for IP for Pod '%s': %v", podName, err)
+	}
+
+	cmd := []string{"ovs-ofctl", "dump-flows", "br-int", fmt.Sprintf("table=10,arp,arp_spa=%s", podIP)}
```

Comment:
is it possible that by that time the GARPS have not been sent yet, since it is an asynchronous operation in the agent?

Reply:
Good catch, I was wondering why the test only failed on Jenkin CI only and it got only 1 ARP packet. Maybe it's because of this. I added a sleep and let's see if it's stable now.